threatx/connector.py

```diff
@@ -0,0 +1,26 @@
+""" Copyright start
```

Comment:
Please replace Copyright lines with the below lines

"""
Copyright start
MIT License
Copyright (c) 2024 Fortinet Inc
Copyright end
"""